### DISCUSSION TITLE
Backdrop around <Marker />

### DISCUSSION THREAD

**Discussion by midrizi** [react-native-maps/react-native-maps#4419] [Q&A]
I'm trying to do something similar to how waze highlights makers. Can this be achieved with `react-native-maps`?

https://user-images.githubusercontent.com/56638730/186259616-2a86fbb5-4e91-4f9f-903f-d5fa91cce0a9.mp4

**Comment by flexbox** [comment]
@midrizi 

You can add a `children` to `<Marker>` like this example

```jsx
        return (
          <Marker
            key={`marker-${index}`}
            coordinate={coordinate}
          >
            <PriceMarker amount={amount} />
          </Marker>
        );
```

<img width="191" alt="image" src="https://user-images.githubusercontent.com/360936/186619381-1148917b-1f64-4e81-a08b-bd5e4e13ebe0.png">

[link to the docs](https://github.com/react-native-maps/react-native-maps/blob/master/docs/marker.md#children-components)

**Reply by midrizi** [reply]
Thanks for the reply @flexbox, 

I am aware that you can add children in `<Marker />` in fact I have an image inside. The problem that I have is with the backdrop, how do I get it to fill the screen?

I thought of using something like https://shopify.github.io/react-native-skia/docs/shapes/polygons/#diffrect which would allow me to create that cutout hole
